libs/designer/package.json

```diff
@@ -3,5 +3,9 @@
   "version": "0.2.68",
   "scripts": {
     "yalcpush": "yalc push ../../dist/libs/designer"
+  },
+  "peerDependencies": {
+    "react": "^16.4.0 || ^17.0.0 || ^18.0.0",
```

Comment:
There were 4 packages that listed react as a dep, is changing these 2 files enough?
They were:
* @microsoft/logic-apps-designer
* @microsoft/designer-ui
* @microsoft-logic-apps/intl
* @microsoft-logic-apps/utils
ex. https://msazure.visualstudio.com/OneAgile/_artifacts/feed/power-platform-ux/Npm/@microsoft-logic-apps%2Futils/overview/0.2.30

Reply:
Need Travis to chime in on this one. I did find a reference in `libs/utils` to React:

![image](https://user-images.githubusercontent.com/1350074/217105702-de221a2e-22db-40f2-a8d9-349b5cbfd03a.png)

Reply:
`libs/intl` has no `react` or `react-dom` references; `libs/utils` has no `react-dom` references.

Therefore I think adding this to `libs/utils/package.json` should be sufficient?

![image](https://user-images.githubusercontent.com/1350074/217107686-cab604c3-52ce-4c7e-b7d6-d29cc326549f.png)

Reply:
Spoke with Travis offline&mdash;we don't want React in `libs/utils`, so he will be removing that reference.